Dockerfile

```diff
@@ -59,7 +59,7 @@ ARG RUNTIME_DEPS="\
     php7-xmlwriter \
     php7-zip \
     sqlite \
-    supervisor \
+    multirun \
```

Comment:
I'm not sure if HumHub has been fully tested with multirun, you should probably run some tests before switching.

Reply:
https://docs.humhub.org/docs/admin/asynchronous-tasks/